File: tree/Tree.go
package main

import "fmt"

type Node struct {
	name string
	children []Node
}

type Tree struct{
	root Node
}

// Adjacency List
type TreeMap map[int][]int
var treeMap = TreeMap{
	0:[]int{1,2},
	1:[]int{3,4},
	2:[]int{5,6},
}
func main() {
	node1 := Node{"1",[]Node{}} 
	node2 := Node{"2",[]Node{node1}}
	tree := Tree{node2}
	fmt.Println(tree)
	fmt.Println(treeMap)



}

File: tree/Tree.py
from typing import Dict, List


class Node:
    def __init__(self, name, children) -> None:
        self.name = name
        self.children = children
class Tree:
    def __init__(self,root) -> None:
        self.root = root
        
# Adjacency List
tree = dict()
tree["0"]=["1","2"]
tree["1"]=["3","4"]
tree["2"]=["5","6"]